docs/source/tutorial_crowdsourcing.md

```diff
@@ -0,0 +1,255 @@
+    
+Advanced Task Techniques
+------------------------
+
+The Mephisto platform allows for a number of advanced customization
+techniques to cover specialized tasks. See the [`bootstrap-chat` README](https://github.com/facebookresearch/Mephisto/blob/master/packages/bootstrap-chat/README.md) for a discussion of how to utilize Bootstrap-based UI components for crowdsourcing tasks.
+
+Running a Task
+--------------
+
+If you have not used Mechanical Turk before, you will need an MTurk
+Requester Account and an AWS account (these are two separate accounts).
+Follow the steps below:
+
+-   Sign up for an AWS account at
+    [aws.amazon.com](https://aws.amazon.com/).
+-   Sign up for an MTurk account at
+    [requester.mturk.com](https://requester.mturk.com/).
+-   Go to the developer tab
+    ([<https://requester.mturk.com/developer>](https://requester.mturk.com/developer))
+    and link your AWS account to your MTurk account (Step 2 on that
+    screen).
+-   MTurk also has a “Sandbox”, which is a test version of the MTurk
+    marketplace. You can use it to test publishing and completing tasks
+    without paying any money. ParlAI supports the Sandbox. To use it, you will need to sign up for a [Sandbox
+    account](http://requestersandbox.mturk.com/), and you will then also
+    need to [link your AWS
+    account](http://requestersandbox.mturk.com/developer) to your
+    Sandbox account. In order to test faster, you will also want to
+    create a [Sandbox Worker account](http://workersandbox.mturk.com/).
+    You can then view tasks that you publish from ParlAI and complete them
+    yourself.
+-   Mephisto's default MTurk functionality requires a free Heroku account,
+    which can be obtained [here](https://signup.heroku.com/). Running
+    any Mephisto MTurk operation will walk you through linking the two.
+    
+To run a crowdsourcing task, launch its run file (typically `run.py`) with the proper flags, using a command like the following:
+
+```bash
+python run.py \
+mephisto.blueprint.num_conversations <num_conversations> \
+mephisto.task.task_reward <reward>
+```
+(Note that the command will launch HITs on the sandbox by default.) For instance, to create 2 conversations for the [QA Data
+Collection](https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing/tasks/qa_data_collection)
+task with a reward of $0.05 per assignment in sandbox mode, run:
+
+```bash
+python parlai/crowdsourcing/tasks/qa_data_collection/run.py \
+mephisto.blueprint.num_conversations 2 \
+mephisto.task.task_reward 0.05
+```
+
+Make sure to test your task in sandbox mode first before pushing it live: see the [crowdsourcing README](https://github.com/facebookresearch/ParlAI/tree/master/parlai/crowdsourcing#running-tasks-live) for how to run a live task.
+
+Additional flags can be used for more specific purposes:
```

Comment:
or perhaps at least include it here

Reply:
Good point, I've included it at the bottom of this section, linking back to the README to explain how to set the YAML file